src/main/java/org/truffleruby/language/objects/shared/SharedObjects.java

```diff
@@ -55,7 +55,7 @@ private static void shareContextRoots(RubyContext context) {
             stack.push(object);
         }
 
-        // Share all named modules and constants (including the shared TOPLEVEL_BINDING)
+        // Share all named modules and constants
```

Comment:
What implications are there for sharing these new binding constants, @eregon? There's one created per top-level now.